RecoVertex/Configuration/python/RecoVertex_EventContent_cff.py

```diff
@@ -45,4 +48,10 @@ def _phase2_tktiming_AddNewContent(mod):
 _phase2_tktiming_AddNewContent(RecoVertexRECO)
```

Comment:
note that both phase2_timing and phase2_timing_layer are present in the MTD case
```
Era_Phase2C4_timing	= (Phase2C4, phase2_timing)
Phase2C4_timing_layer_bar	= (Phase2C4_timing, phase2_timing_layer, phase2_timing_layer_bar)
```

so, the event content will include both for the MTD TDR setup.
Is this actually desired?

Reply:
if decoupled version is needed, IIRC the syntax is ```( phase2_timing & ~ phase2_timing_layer).toModify(```

Reply:
Yes, this is intentional